recipes/conversation_view/message_input/message_input.vue

```diff
@@ -554,6 +555,12 @@ export default {
     },
   },
 
+  watch: {
+    modelValue (newValue) {
+      this.inputValue = newValue;
```

Comment:
When we change `this.inputValue` internally such as in onSelectEmoji, does that need to fire an `update:modelValue` event to keep modelValue in sync or is this done some other way?

Reply:
since we use v-model on richTextEditor: (https://github.com/dialpad/dialtone-vue/pull/1058/files#diff-62664268bb3876ed01b9494168e2b23396f4748e4a4f36c1ed22a5ca075b57b1R5), changing inputValue should do both i guess, firing the event and also changing the prop value. it works without sending that event. Or are you suggesting to propagate the value upwards? That anyway happens with input event on ever change.

Reply:
This is sort of off the top of my head so let me know if I have not recognized some issue with this, but could we get rid of this internal inputValue completely?

here's what I'm thinking:

1. pass modelValue directly into dt-rich-text-editor
2. remove this watch
3. in `onSelectEmoji` instead of setting `this.inputValue = this.inputValue + emoji.shortname;` do `emit('update:modelValue', this.modelValue + emoji.shortname)`

Would this work or am I crazy?

Reply:
@superfreddialpad tagging fred here since he wanted to think about nuking the entire prop and have a ref exposed to set content. 
Meanwhile i will try what you mentioned brad!

Reply:
@braddialpad vue doesnt allow it with unexpected mutation of prop error. In rich text editor its not directly used as a prop so its fine. but in this component, we need to send it to rich text editor which poses a problem.

Reply:
Hmm where are we still mutating the prop after making this change?

Reply:
I think the watch should be fine? @sanjeevdialpad lets meet tomorrow and figure this out. Hopefully we can just make it work with `v-model` (modelValue)